<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>720</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Simple VIdeo Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="2" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_8"/>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="btnFullscreen">
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Show fullscreen</string>
      </property>
      <property name="whatsThis">
       <string>Show video in fullscreen</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../simple-media-player-old/trunk/MyPlayer/share/icons/view-fullscreen.png</normaloff>../../../../../simple-media-player-old/trunk/MyPlayer/share/icons/view-fullscreen.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="btnShowPlayList">
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Show/Hide Playlist</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../simple-media-player-old/trunk/MyPlayer/share/icons/view-media-playlist.png</normaloff>../../../../../simple-media-player-old/trunk/MyPlayer/share/icons/view-media-playlist.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QPushButton" name="btnAbout">
      <property name="maximumSize">
       <size>
        <width>48</width>
        <height>24</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>About</string>
      </property>
      <property name="whatsThis">
       <string>About this program</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../simple-media-player-old/trunk/MyPlayer/share/icons/help-about.png</normaloff>../../../../../simple-media-player-old/trunk/MyPlayer/share/icons/help-about.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="8">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QWebView" name="playerView">
          <property name="url">
           <url>
            <string>https://www.youtube.com/</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QWebView" name="videoList">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btnNext">
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Next</string>
      </property>
      <property name="whatsThis">
       <string>Play next media</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../simple-media-player-old/trunk/MyPlayer/share/icons/media-skip-forward.png</normaloff>../../../../../simple-media-player-old/trunk/MyPlayer/share/icons/media-skip-forward.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnPrevious">
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Previous</string>
      </property>
      <property name="whatsThis">
       <string>Play previous media</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../simple-media-player-old/trunk/MyPlayer/share/icons/media-skip-backward.png</normaloff>../../../../../simple-media-player-old/trunk/MyPlayer/share/icons/media-skip-backward.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="showVideoPage">
      <property name="text">
       <string>Show Video Page</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="showSearchPage">
      <property name="text">
       <string>Show Search Result</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dckPlayList">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>176</width>
     <height>411</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>524287</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Choose a tab</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QWidget" name="wdgPlayList" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnAddMedia">
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>24</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Add media</string>
          </property>
          <property name="whatsThis">
           <string>Add media to playlist</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icons/list-add.png</normaloff>../icons/list-add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnShufflePlayList">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Shuffle</string>
          </property>
          <property name="whatsThis">
           <string>Shuffle playlist</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icons/media-playlist-shuffle.png</normaloff>../icons/media-playlist-shuffle.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRepeatPlayList">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Repeat</string>
          </property>
          <property name="whatsThis">
           <string>Repeat playlist</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icons/media-playlist-repeat.png</normaloff>../icons/media-playlist-repeat.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemoveMedia">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Remove media</string>
          </property>
          <property name="whatsThis">
           <string>Remove media from playlist</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icons/list-remove.png</normaloff>../icons/list-remove.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClearPlayList">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Clear playlist</string>
          </property>
          <property name="whatsThis">
           <string>Clear playlist</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icons/edit-clear.png</normaloff>../icons/edit-clear.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tbPlaylist">
        <attribute name="title">
         <string>Play list</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QListWidget" name="lswPlaylist">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>2</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDrop</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tbSch">
        <attribute name="title">
         <string>Search</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Sort by</string>
               </property>
               <property name="buddy">
                <cstring>sortbyButtonGroup</cstring>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QRadioButton" name="rdbRelevance">
                 <property name="text">
                  <string>Relevance</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">sortbyButtonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdbPublished">
                 <property name="text">
                  <string>Published date</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">sortbyButtonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdbView">
                 <property name="text">
                  <string>View count</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">sortbyButtonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdbRating">
                 <property name="text">
                  <string>Rating</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">sortbyButtonGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Safe search</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QRadioButton" name="rdbSafeNone">
                 <property name="text">
                  <string>None</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">safeSearchButtonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdbSafeModerate">
                 <property name="text">
                  <string>Moderate</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">safeSearchButtonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdbSafeStrict">
                 <property name="text">
                  <string>Strict</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">safeSearchButtonGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Search term</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditSearch">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnSearchVideo">
           <property name="text">
            <string>Seach</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tbFeeds">
        <attribute name="title">
         <string>Feeds</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Standard feeds</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="btnMostViewed">
               <property name="text">
                <string>Most Viewed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnTopRated">
               <property name="text">
                <string>Top Rated</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRecentlyFeatured">
               <property name="text">
                <string>Recently Featured</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnMostDiscussed">
               <property name="text">
                <string>Most Discussed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnTopFavorites">
               <property name="text">
                <string>Top Favorites</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnMostLinked">
               <property name="text">
                <string>Most Linked</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnMostResponded">
               <property name="text">
                <string>Most Responded</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnMostRecent">
               <property name="text">
                <string>Most Recent</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Feed by user</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="lineeditUserFeed"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tbDownload">
        <attribute name="title">
         <string>Download</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rdbRelevance</tabstop>
  <tabstop>rdbPublished</tabstop>
  <tabstop>rdbView</tabstop>
  <tabstop>rdbRating</tabstop>
  <tabstop>rdbSafeNone</tabstop>
  <tabstop>rdbSafeModerate</tabstop>
  <tabstop>rdbSafeStrict</tabstop>
  <tabstop>lineEditSearch</tabstop>
  <tabstop>lswPlaylist</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>on_btnPlayPause_clicked()</slot>
  <slot>on_btnPrevious_clicked()</slot>
  <slot>on_btnNext_clicked()</slot>
  <slot>on_btnFullscreen_clicked()</slot>
  <slot>on_cbxVideoAspect_currentIndexChanged(int)</slot>
  <slot>on_btnAbout_clicked()</slot>
  <slot>on_btnShufflePlayList_clicked()</slot>
  <slot>on_btnRepeatPlayList_clicked()</slot>
  <slot>on_btnRemoveMedia_clicked()</slot>
  <slot>on_btnClearPlayList_clicked()</slot>
  <slot>on_lswPlaylist_doubleClicked(QModelIndex)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="sortbyButtonGroup"/>
  <buttongroup name="safeSearchButtonGroup"/>
 </buttongroups>
</ui>
